src/__tests__/OrsUtil.cy.js

```diff
@@ -0,0 +1,81 @@
+import OrsUtil from '../OrsUtil.js'
+import Constants from "../constants.js"
+
+const orsUtil = new OrsUtil()
+
+describe('Test Utils', () => {
+  it('requestArgs are filled with defaultArgs', () => {
+    let response = orsUtil.
+      fillArgs({
+        a: 'a',
+        b: 'b',
+        c: 'c'
+      },
+    {
+        d: 'd',
+        b: 'b2'
+      })
+      expect(response).to.include({ b: 'b2' })
+  })
+
+  it('args are saved to cache', () => {
+    let response = orsUtil.
+      saveArgsToCache({
+      [Constants.propNames.host]: 'host',
+      [Constants.propNames.profile]: 'profile',
+      [Constants.propNames.service]: 'service'
+      })
+    expect(response).to.include({
+      host: 'host',
+      profile: 'profile',
+      service: 'service'
+    })
+  })
+
+  it('properties are removed', () => {
+    let response = orsUtil.
+      prepareRequest({})

```

Comment:
This test does not really test the function, as it runs on an empty object.
We need an object with properties here that are actually removed.